frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,57 @@
+import { Column, useTable } from 'react-table';
+import { FC } from 'common/types/types';
+import styles from './table/styles.module.scss';
+
+type Props<Data extends Record<string, unknown>> = {
+  columns: Column<Data>[];
+  data: readonly Data[];
+};
+const Table: FC<Props<Record<string, unknown>>> = ({ columns, data }) => {
+  const tableInstance = useTable({
+    columns: columns as Column<Record<string, string>>[],
+    data: data as Record<string, string>[],
```

Comment:
I think you could provide generic typing from data props for useTable instead of using type cast